src/quo/components/text_combinations/page_top/view.cljs

```diff
@@ -37,17 +37,17 @@
   [{:keys        [title title-accessibility-label input counter-top counter-bottom
                   title-right title-right-props]
     avatar-props :avatar}]
-  (let [avatar-props (assoc avatar-props :size :size-32)
-        title-props  (assoc title-right-props
-                            :title               title
-                            :right               title-right
-                            :accessibility-label title-accessibility-label)]
+  (let [title-props (assoc title-right-props
+                           :title               title
+                           :right               title-right
+                           :accessibility-label title-accessibility-label)]
     [rn/view {:style style/header}
      [rn/view {:style style/header-title}
       (when avatar-props
-        (if (:group? avatar-props)
-          [group-avatar/view avatar-props]
-          [channel-avatar/view avatar-props]))
+        (let [avatar-props (assoc avatar-props :size :size-32)]
```

Comment:
it might be worth checking this doesn't break on other pages too as this component is used on a lot of screens!

Reply:
Looks good to me

https://github.com/status-im/status-mobile/assets/10757633/8937b506-7cf9-4722-bd63-a43be1da6389

Reply:
thanks @Rende11!